content/docs/command-reference/remote/modify.md

```diff
@@ -758,6 +758,43 @@ more information.
 
 </details>
 
+<details>
+
+### Click for OSF
+
+- `url` - remote location:
+
+  ```dvc
+  $ dvc remote modify myremote url \
+        osf://odf.io/path/to/dir
```

Comment:
Looks like most/all of the code blocks are not closed like this:
````suggestion
        osf://odf.io/path/to/dir
    ```
````

Reply:
p.s. probably this code fits in a single line BTW, no need for `\` and line 769 above.

Reply:
I fixed all this.